src/adapters/libuv.h

```diff
@@ -191,7 +191,7 @@ uvAsyncAttach(natsLibuvEvents *nle)
         s = NATS_NO_MEMORY;
 
     if ((s == NATS_OK)
-        && (uv_poll_init_socket(nle->loop, nle->handle, nle->socket) != 0))
```

Comment:
It actually compiles fine without any warning, but I will change the `(*natsEvLoop_Attach)` function prototype to use `natsSock` which represents properly a socket based on the platform it is built on (master is in a new release 1.7.0 schedule, so I am ok to change `nats.h` content). I will do so in upcoming PR.